### PR TITLE
bfulgham/merge webkit 255154

### PR DESCRIPTION
**Pull request by brentfulgham**
Merge fuzzy settings for Webkit (GTK/iOS/macOS/WPE) - Bug 255154
https://bugs.webkit.org/show_bug.cgi?id=255154

### CODE REVIEW

File: css/css-backgrounds/border-image-repeat-space-8.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
 <link rel="help" href="https://www.w3.org/TR/css3-background/#the-border-image-repeat">
 <link rel="match" href="border-image-repeat-space-8-ref.html">
 <meta name="assert" content="An explicit 'border-image-width' which is exactly the same size as 'border-width' should render the same as 'border-image-width:1' (the initial value).">
+<meta name="fuzzy" content="maxDifference=110; totalPixels=10540" />
```
Comment (dbaron) [inline css/css-backgrounds/border-image-repeat-space-8.html:8]: This one should have `0-` in both ranges.

File: css/css-backgrounds/border-image-space-001.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <link rel="author" title="Levi Weintraub" href="mailto:leviw@chromium.org">
   <link rel="help" href="http://www.w3.org/TR/css3-background/#the-border-image-repeat">
   <meta name="assert" content="border-image-repeat: space property spaces out background image that doesn't fit an even number of times.">
-  <meta name="fuzzy" content="maxDifference=0-80;totalPixels=0-1728">
+  <meta name="fuzzy" content="maxDifference=0-255; totalPixels=0-3895">
```
Comment (nt1m) [inline css/css-backgrounds/border-image-space-001.html:9]: This is suspicious, it's suggesting 3895 pixels can have completely different color (0-255 is the max range).

Can we revert https://commits.webkit.org/262031@main and this change from the PR?

**Review by nt1m — APPROVED** [review]

**Review by dbaron — CHANGES_REQUESTED** [review]
Mostly looks fine, I suppose, though some of the numbers are a bit large.  Did you check that the images at least appear to still be passing the point of the test?

One change request, though.

File: css/css-backgrounds/border-image-space-001.html (hunk 2 above)
Comment (brentfulgham) [reply css/css-backgrounds/border-image-space-001.html:9]: Yes -- I'll do a separate change for 262031@main in WebKit, and remove this change here.

File: css/css-backgrounds/border-image-repeat-space-8.html (hunk 1 above)
Comment (brentfulgham) [reply css/css-backgrounds/border-image-repeat-space-8.html]: Will do!

**Comment by brentfulgham** [comment]
> Mostly looks fine, I suppose, though some of the numbers are a bit large. Did you check that the images at least appear to still be passing the point of the test?

The images looked correct to me, but it's possible I missed subtle variations in shading or gradients.

**Pull request merged**